### PR TITLE
Fix/podspec - Fix quotes issues

### PR DESCRIPTION
**Pull request by daviddgl**
When we were building the app we got the following issue:
```
/Users/builder/programs/flutter_1_22_6/.pub-cache/hosted/pub.dartlang.org/screen_lock_check-1.0.0/ios/Classes/ScreenLockCheckPlugin.m:19:43: error: non-ASCII characters are not allowed outside of literals and identifiers
         result([FlutterError errorWithCode: @“UNAVAILABLE” message: @“Local authentication context is unavailable” details: nil]);
```

The issue was in the quotes, they were incorrect.
Tying to find the issue I also improve some info and the example project.

### CODE REVIEW

File: ios/Classes/ScreenLockCheckPlugin.m [hunk 1]
```diff
@@ -16,7 +16,9 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
    NSError *error;
    BOOL isLockScreenEnabled = [context canEvaluatePolicy:LAPolicyDeviceOwnerAuthentication error:&error];
    if (error != nil) {
-     result([FlutterError errorWithCode: @“UNAVAILABLE” message: @“Local authentication context is unavailable” details: nil]);
+       result([FlutterError errorWithCode:@"UNAVAILABLE"
```
Comment (daviddgl) [inline ios/Classes/ScreenLockCheckPlugin.m:19]: As you can notice these quotes were not the ones iOS liked.

File: ios/screen_lock_check.podspec [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
-  s.source           = { :path => '.' }
+  s.homepage         = 'https://www.koahealth.com/'
+  s.license          = { :type => 'MIT', :file => '../LICENSE' }
+  s.author           = { 'Your Company' => 'koa-engineering-all@koahealth.com' }
+  s.source           = { :http => 'https://github.com/koa-health/screen_lock_check/tree/master' }
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.platform = :ios, '8.0'
+  s.platform = :ios, '9.0'
```
Comment (daviddgl) [inline ios/screen_lock_check.podspec:19]: Move to ios version 9

File: example/ios/Podfile.lock [hunk 3]
```diff
@@ -0,0 +1,22 @@
+PODS:
+  - Flutter (1.0.0)
+  - screen_lock_check (0.0.1):
```
Comment (racloubon) [inline example/ios/Podfile.lock:3]: Should this not be `screen_lock_check 1.0.0` ?

File: ios/Classes/ScreenLockCheckPlugin.m (hunk 1 above)
Comment (racloubon) [reply ios/Classes/ScreenLockCheckPlugin.m:19]: And this is what happens when you copy and paste from the internet...

**Review by racloubon — APPROVED** [review]
Thanks!

**Pull request merged**